robot/Crisis-Management/tests/browser/t5_user_able_to_update_availibility_staff.robot

```diff
@@ -0,0 +1,60 @@
+    ...                                   Available__c=Assigned      
+    ...                                   Staff__r.Email=jjoseph@salesforce.com
+    ...                                   Care_Facility__r.Name=Julian Account
+    Store Session Record                  Assignment__c                    
+    ...                                   ${assignment_assigned}[0][Id]
+    ${num_of_assiangment_assigned}=       Get Length	        
+    ...                                   ${assignment_assigned}
+    Should Be Equal As Integers	          ${num_of_assiangment_assigned}	        1
+
+Verify There Is 1 Unavailibility
+    @{assignment_not_available}=          Salesforce Query  Assignment__c   
+    ...                                   select=Id   
+    ...                                   Available__c=Not Available     
+    ...                                   Staff__r.Email=jjoseph@salesforce.com
+    Store Session Record                  Assignment__c                    
+    ...                                   ${assignment_not_available}[0][Id]
+    ${num_of_assiangment_not_available}=  Get Length	        
+    ...                                   ${assignment_not_available}
+    Should Be Equal As Integers	          ${num_of_assiangment_not_available}	   1
+
+Verify There Are 14 Availibilities
+    Wait Until Element Is Visible   //*[@class="cmp-availability-date-wrap"]  #Availibility Buttons
+    ${count} =                      SeleniumLibrary.Get Element Count   //*[@class="cmp-availability-date-wrap"]  #Count Buttons
+    Should Be Equal As Strings      ${count}                            14
+
+Set 1st Availibility to Not Available
+    Click Element                           //*/li[1]/c-staff-availability-item/div/div/div[2]/div/div[@data-available="Available"]  #1st Availibility Button
+    Click Element                           //*/span[1]/label/span[@class="slds-radio_faux"]  #Not Available Option
+    Click Element                           //*//div/a[2]/span[@class="slds-m-left_x-small"]  #OK Button
+    SeleniumLibrary.Element Text Should Be  //*/li[1]/c-staff-availability-item/div/div/div[2]/div/div[1][@class="cmp-availability"]  Not Available  #1st Availibility Button
+
+Set 4th Availibility to Availible at Julian Account
+    Click Element                           //*/li[4]/c-staff-availability-item/div/div/div[2]/div/div[@data-available="Available"]  #4th Availibility Button
+    Click Element                           //*/li[4]/c-staff-availability-item/div/div[2]/div[1]/div/lightning-radio-group/fieldset/div/div/span[3]/label/span[@class="slds-radio_faux"]  #Assigned Option
+    Click Element                           //*/li[4]/c-staff-availability-item/div/div[2]/div[2]/div/a[2]/span[@class="slds-m-left_x-small"]  #OK Button
+    SeleniumLibrary.Element Text Should Be  //*/li[4]/c-staff-availability-item/div/div/div[1]/div[3]/span[@class="cmp-facility-name"]  Julian Account  #Account
```

Comment:
Please move all the locators to locators.py or locators_47.py file. The locators mentioned here might not be reliable as the indices of these might keep changing with the change in application. We can go over in tomorrow’s call on this if you are interested. For Validations, it is good to practice to wrap it in a function and refer the function/keyword in the robot file.

Reply:
@sgovindankutty yeah would love to discuss these best practices during the meeting :-)